googletest/cmake/internal_utils.cmake

```diff
@@ -78,10 +78,11 @@ macro(config_compiler_and_linker)
     # http://stackoverflow.com/questions/3232669 explains the issue.
     set(cxx_base_flags "${cxx_base_flags} -wd4702")
   elseif (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
-    set(cxx_base_flags "-Wall -Wshadow -Werror -Wno-error=sign-conversion")
+    set(cxx_base_flags "-Wall -Wshadow -Werror -Wno-error=sign-conversion -std=c++11 -frtti")
     set(cxx_exception_flags "-fexceptions")
     set(cxx_no_exception_flags "-fno-exceptions")
     set(cxx_strict_flags "-W -Wpointer-arith -Wreturn-type -Wcast-qual -Wwrite-strings -Wswitch -Wunused-parameter -Wcast-align -Wchar-subscripts -Winline -Wredundant-decls")
+    set(cxx_no_rtti_flags "-fno-rtti")
```

Comment:
Why don't we need to pass `-DGTEST_HAS_RTTI=0`  like we do when using GCC?

Reply:
Because the compiler (clang) reports whether or not it's disabled and googletest tests for it: https://github.com/google/googletest/blob/master/googletest/include/gtest/internal/gtest-port.h#L519